schematic/manifest/commands.py

```diff
@@ -153,7 +153,7 @@ def get_manifest(
 
     # Generate graph
     logger.info("Generating data model graph.")
-    graph_data_model = data_model_grapher.generate_data_model_graph()
+    graph_data_model = data_model_grapher.graph
```

Comment:
@mialy-defelice I already made this change. Is this what you were referring to in your comment?

Reply:
Ah yes!